.github/workflows/prepare-uv.yml

```
@@ -0,0 +1,64 @@
+---
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)